charts/incubator/tt-rss/values.yaml

```diff
@@ -179,3 +187,80 @@ service:
         targetSelector: main
         port: 9000
         targetPort: 9000
+configmap:
+  config:
+    enabled: true
+    data:
+      nginx.conf.template: |-
+        worker_processes auto;
+        pid /var/run/nginx.pid;
+
```

Comment:
Lets remove empty lines here
(Mostly because it consumes space when stored in a configmap, and **current** helm versions dont handle the case that you exceed the limit!)